.travis.yml

```diff
@@ -25,20 +25,6 @@ _osx: &osx
 
 jobs:
   include:
-# Build
```

Comment:
It's kinda redundant if we're testing each rule though, since the rule will need to be built for the tests anyway. There's only one rule in this rule set, and we don't anticipate there being any more in the near future (and if we do add more, we need to add tests for them anyway)